src/ObjectGraph.es6.js

```diff
@@ -182,21 +184,25 @@ ObjectGraph.prototype.rewriteName = function(name) {
 
 // Visit the prototype of o, given its dataMap.
 ObjectGraph.prototype.visitPrototype = function(o, dataMap) {
+  if ( this.protos[uid.getId(o)] !== undefined ) {
+    console.log(uid.getId(o));
```

Comment:
... weren't we going to add a `maybeSkip` to this implementation? https://github.com/mdittmer/object-graph-js/pull/11#discussion-diff-104459768R253